docs/concepts/containers/images.md

```diff
@@ -193,7 +193,7 @@ template needs to include the `.docker/config.json` or mount a drive that contai
 All pods will have read access to images in any private registry once private
 registry keys are added to the `.docker/config.json`.
 
-**This was tested with a private docker repository as of 26 June with Kubernetes version v0.19.3.
+**This was tested with a private docker repository as of 26 June with Kubernetes version v1.9.3.
```

Comment:
So why did this have to change? Was the original version number incorrect? Would this sample not work with both 0.19.3 and 1.9.3?   The way this line is phrased where it says "as of 26 June"  seems awkward, unnecessary, and frankly vague.  I know that portion was already there but if we are going to update this I think we should update it properly.  @zacharysarah I would appreciate your input on this as well.

Reply:
Yes  26 june awkward...

I tested v1.9.3 and it's working...

I think version number is incorrect .. is 0.19.3 can accessible ? branch tag start v1.6.7

Reply:
Let's remove this line entirely. It's not helping users to know that a deprecated version of Kubernetes was tested on a specific date (without a year!).

Reply:
@zacharysarah @steveperry-53 Any concerns with me just editing the commit and removing the line entirely?  Just checking since @steveperry-53 added an open issue label and I wasn't sure why.